### DISCUSSION TITLE
Different behavior between map and waypoints

### DISCUSSION THREAD

**Discussion by guerrillian** [Courseplay/Courseplay_FS22#2625] [Q&A]
The map shows a straight line in the curve, but the course is working fine as you can see in the 2nd image.

![Screenshot_4](https://github.com/Courseplay/Courseplay_FS22/assets/128705907/1b9f659b-acec-44be-a7f4-dde20abfae81)
![Screenshot_5](https://github.com/Courseplay/Courseplay_FS22/assets/128705907/5285d9b8-547a-4213-b282-1e90484152b1)

**Comment by Tensuko** [comment]
Is this a generated course?

**Reply by guerrillian** [reply]
I designed the curve in the editor.

**Reply by Tensuko** [reply]
Maybe just not updated then, not a big deal.
